README.md

```diff
@@ -1,44 +1,4 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+* Mars Mission *
 
-## Available Scripts
-
-In the project directory, you can run:
-
-### `yarn start`
-
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
-
-### `yarn test`
-
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `yarn build`
-
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `yarn eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+Accessing API Key:
+ You can access the NASA api key with: process.env.REACT_APP_NASA_API_KEY
```

Comment:
The most important bits of information here is what the dev needs to do before the app will work for them!

Think of it as an easy to follow guide for getting a new developer up to speed with the project.

The `process.env` stuff we can probably assume a developer has seen before, as its just a standard feature of node.

Could we have something like
````Markdown
# Mars Mission

An educational website for inspiring interest in Mars.

You can view it at https://techswitch-mars-mission.herokuapp.com/
(or our staging environment at https://techswitch-mars-mission-stage.herokuapp.com/)

## Setup
The app is a react application built using Create React App.

You should be able to start it using 
`npm start`

And run the tests using
`npm test`

### Environment Variables
We use `.env` files to manage our environment variables. This should live in the root of your application.
The environment variables used are:

*REACT_APP_NASA_API_KEY:* Your API for the NASA API (obtained from https://api.nasa.gov/)
```